modules/dnn/test/test_tf_importer.cpp

```diff
@@ -69,20 +69,61 @@ TEST(Test_TensorFlow, inception_accuracy)
     normAssert(ref, out);
 }
 
+static bool readFileInMemory(const string& filename, char** data, size_t* len)
```

Comment:
@r2d3, do you mind if we return a content by `std::string`? In example,
```cpp
static bool readFileInMemory(const string& filename, std::string* content) {
  std::ifstream ifs(filename.c_str());
  CV_Assert(ifs.is_open());

  ifs.seekg(0, std::ios::end);
  content->resize(ifs.tellg());
  ifs.seekg(0, std::ios::beg);
  ifs.read(&content->operator[](0), content->size());
}
```

Reply:
No, I was thinking about it as it is how I do file loading in personal tests

Reply:
Done